mymake.cpp

```diff
@@ -84,6 +83,7 @@ bool file_exists(string fname) {
   }
   
 int main(int argc, char **argv) {
+  setvbuf(stdout, NULL, _IONBF, 0); // this should help with responsiveness of the real-time CI logs
```

Comment:
As long as you're touching it anyway, how about `s/NULL/nullptr/`? :)

Reply:
Aye, I forgot that we're in C++ land here.